pkgs/development/compilers/flasm/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, unzip, bison, flex, gperf, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "flasm";
+  version = "1.64";
+
+  src = fetchurl {
+    url = "https://www.nowrap.de/download/flasm16src.zip";
+    sha256 = "0n7sqv0b4f3wjfrhlv9d1ynza1chcmnrqxvpp0b3dh68ridnjdxv";
+  };
+
+  unpackPhase = ''
+    unzip $src -d src
+    cd src
+  '';
+
+  # Make build reproducible
+  patchPhase = ''
+    substituteInPlace flasm.c --replace 'printf("\nFlasm %s build %s", FLASM_VERSION, __DATE__);' 'printf("\nFlasm %s", FLASM_VERSION);'
+  '';
+
+  makeFlags = [ "CC=cc" ];
```

Comment:
I guess you don't need it, `cc` is just a symlink to `gcc`.

Reply:
This is needed for darwin build

Reply:
Can you maybe add a comment to clarify that?